modules/installation-configuration-parameters.adoc

```diff
@@ -1558,6 +1558,11 @@ Additional {ibmpowerProductName} Virtual Server configuration parameters are des
 1. Whether you define an existing resource group, or if the installer creates one, determines how the resource group is treated when the cluster is uninstalled. If you define a resource group, the installer removes all of the installer-provisioned resources, but leaves the resource group alone; if a resource group is created as part of the installation, the installer removes all of the installer provisioned resources and the resource group.
 2. To determine which profile best meets your needs, see https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui[Instance Profiles] in the IBM documentation.
 --
+
+[NOTE]
```

Comment:
I don't see a preview for this, but the text looks OK.

Reply:
@kcarmichael08 here is the preview link for the NOTE: https://65856--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installation-config-parameters-ibm-power-vs#installation-configuration-parameters-additional-ibm-cloud_installation-config-parameters-ibm-power-vs